packages/application-generic/src/factories/mail/handlers/plunk.handler.ts

```diff
@@ -0,0 +1,18 @@
+import { ChannelTypeEnum, ICredentials } from '@novu/shared';
+import { PlunkEmailProvider } from '@novu/plunk';
+import { BaseHandler } from './base.handler';
+
+export class PlunkHandler extends BaseHandler {
+  constructor() {
+    super('plunk', ChannelTypeEnum.EMAIL);
+  }
+
+  buildProvider(credentials: ICredentials, from?: string) {
+    const config: { apiKey: string; from: string } = {
+      apiKey: credentials.apiKey,
+      from,
+    };
+
+    this.provider = new PlunkEmailProvider(config);
```

Comment:
`from` is not being used later in the provider. It is being taken from `options`. Does Plunk do not need to set a `from` email?
Also sender name that is mandatory for the email config base is not being passed in any way.

Reply:
The `from` and `name` parameter are optional with Plunk. Our users verify their domain in the dashboard and select a default from address. The `from` parameter in our API can be used to overwrite the default value.

Reply:
> The `from` and `name` parameter are optional with Plunk. Our users verify their domain in the dashboard and select a default from address. The `from` parameter in our API can be used to overwrite the default value.

Considering in our system `from` and `senderName` are mandatory parameters to add in our integration our users are going to expect that those values are going to be the used ones if no overrides are placed. So from a point of view of having less cognitive burden I'd pass them in the constructor of the provider and do something like we do in the Sendgrid provider and specify the fallbacks in the usage:
https://github.com/novuhq/novu/blob/a4a0b9681ff89b7d1e16d51b7c9cb773c65d573e/providers/sendgrid/src/lib/sendgrid.provider.ts#L68
But after reviewing our SendMessageEmail use case I reckon we sort that out already so we could leave the provider like this. We can fix it up if any weird behaviour happens in the future. 👍🏻